src/utilities/effects.ts

```diff
@@ -3793,20 +3793,18 @@ const effects = {
     }
   },
 
-  async updateActivityPreset(
-    id: ActivityPresetId,
-    updatedActivityPreset: ActivityPresetSetInput,
-    user: User | null,
-  ): Promise<void> {
+  async updateActivityPreset(updatedActivityPreset: ActivityPresetSetInput, user: User | null): Promise<void> {
     try {
       if (!queryPermissions.UPDATE_ACTIVITY_PRESET(user, updatedActivityPreset)) {
         throwPermissionError('update this activity preset');
       }
-      const { id: _id, ...restOfPresetPayload } = updatedActivityPreset;
+
+      const { id: id, ...restOfPresetPayload } = updatedActivityPreset;
```

Comment:
can you change to ``` { id, ...restOfPresetPayload }```?  It's just shorthand.

Reply:
Done!